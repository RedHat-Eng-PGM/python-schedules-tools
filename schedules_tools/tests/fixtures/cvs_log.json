[
    {
        "author": "user5",
        "datetime": "2013-08-22T15:24:33Z",
        "message": "rename file",
        "revision": "1.1"
    },
    {
        "author": "anotherbody",
        "datetime": "2013-08-26T13:59:13Z",
        "message": "another comment on this line",
        "revision": "1.2"
    },
    {
        "author": "user123",
        "datetime": "2013-08-28T10:04:17Z",
        "message": "new change",
        "revision": "1.2.2.1"
    },
    {
        "author": "anybody",
        "datetime": "2014-01-09T16:51:09Z",
        "message": "Update date\nsome so long commit message\nsplitted to multiple\nlines",
        "revision": "1.3"
    },
    {
        "author": "nobody",
        "datetime": "2016-10-13T13:48:18Z",
        "message": "cleanup, no schedule changes",
        "revision": "1.4"
    }
]
