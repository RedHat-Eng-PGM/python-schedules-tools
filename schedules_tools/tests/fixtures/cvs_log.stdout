
RCS file: /cvs/some/path/schedule/schedule-0,v
Working file: schedule/schedule-0.tjp
head: 1.4
branch:
locks: strict
access list:
symbolic names:
	stage_root_7: 1.3
	revert1: 1.2
	stage_root_6: 1.2
	stage: 1.2.0.2
	stage_root_5: 1.2
keyword substitution: kv
total revisions: 5;	selected revisions: 5
description:
----------------------------
revision 1.4
date: 2016/10/13 13:48:18;  author: nobody;  state: Exp;  lines: +4 -9
cleanup, no schedule changes
----------------------------
revision 1.3
date: 2014/01/09 16:51:09;  author: anybody;  state: Exp;  lines: +10 -4
Update date
some so long commit message
splitted to multiple
lines
----------------------------
revision 1.2
date: 2013/08/26 13:59:13;  author: anotherbody;  state: Exp;  lines: +152 -18
branches:  1.2.2;
another comment on this line
----------------------------
revision 1.1
date: 2013/08/22 15:24:33;  author: user5;  state: Exp;
rename file
----------------------------
revision 1.2.2.1
date: 2013/08/28 10:04:17;  author: user123;  state: Exp;  lines: +1 -1
new change
=============================================================================
